src/krylov/pcg.c

```diff
@@ -690,7 +690,7 @@ hypre_PCGSolve( void *pcg_vdata,
                HYPRE_Real r2ob2;
                /* v = C*s = C*(r_old-r_new) */
                (*(pcg_functions->ClearVector))(v);
-               precond(precond_data, A, s, v);
+               precond(precond_data, A, s, v);      // RDF: Should this use precond_Mat?
```

Comment:
Yes, that should be precond_Mat instead of A. Apparently, we missed this since it is in a spot that is not used that often.